tests/server/TestConfig.js

```diff
@@ -1,69 +1,6 @@
 import { assert } from 'chai';
-import config from 'config';
 import requireUncached from 'require-uncached';
 
-import { getClientConfig } from 'core/utils';
-
-const appsList = config.get('validAppNames');
-
-
-// eslint-disable-next-line no-restricted-syntax
-for (const appName of appsList) {
-  describe(`Config for ${appName}`, () => {
-    let oldNodeEnv;
-
-    before(() => {
-      oldNodeEnv = process.env.NODE_APP_INSTANCE;
-    });
-
-    beforeEach(() => {
-      process.env.NODE_APP_INSTANCE = appName;
-      process.env.NODE_ENV = 'production';
```

Comment:
They test the configs as they are? I'd say no... if we commit bad prod configs we'll see it on stage or something... and we rarely edit them. And that sounds like a fragile and easily fooled test anyway. I'd say it's fine to ditch 👍